changelogs/fragments/70704-void-linux-reboot.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
```

Comment:
```suggestion
bugfixes:
```

Reply:
We discussed this today during triage and this can be considered a bugfix, which means it can be backported rather than having to wait for the 2.11 release. Please update the changelog and I'll merge this.